### PR TITLE
[PRP-278] Update infra to support participant portal doc upload

### PR DESCRIPTION
**Pull request by rocketnova**
## Ticket

https://wicmtdp.atlassian.net/browse/PRP-278

- Add new environment variables needed for doc upload in the participant portal
- Grant participant ECS task IAM higher level permissions to s3 bucket
- Set participant max file limit and max file size lower in dev and staging
- Always define a task IAM role for ECS tasks, even though sometimes they won't need it
- Attach the s3 permissions to the task role, not the task executor role

These changes have been deployed to the dev environment. To test:
- Navigate to dev.wic-services.org/gallatin/recertify/changes and upload a document
- The upload should work and not error in the browser
- Go to the s3 bucket and see your document has been uploaded

### CODE REVIEW

File: infra/app/envs/staging/main.tf [hunk 1]
```diff
@@ -50,13 +50,18 @@ module "project_config" {
...
-  source                = "../../env-template"
-  environment_name      = local.environment_name
-  participant_image_tag = var.participant_image_tag
-  staff_image_tag       = var.staff_image_tag
+  source           = "../../env-template"
+  environment_name = local.environment_name
+
+  participant_image_tag             = var.participant_image_tag
+  participant_url                   = "${local.environment_name}.wic-services.org"
+  participant_max_upload_size_bytes = "5242880"
```
Comment (microwavenby) [inline infra/app/envs/staging/main.tf]: Should we set these to prod numbers, just so there's a non-prod with the same settings?

File: infra/app/env-template/variables.tf [hunk 2]
```diff
@@ -1,24 +1,42 @@
...
+  description = "URL to access the application"
 }
 
-variable "analytics_image_tag" {
+variable "participant_max_upload_size_bytes" {
   type        = string
-  description = "Image tag to deploy to the environment for the analytics service"
-  default     = "latest"
+  description = "The maximum allowed size of a single file upload"
+  default     = "26_214_400"
```
Comment (microwavenby) [inline infra/app/env-template/variables.tf]: I actually don't know - does Javascript parseInt("26_214_400") as `26214400` ? (I know that `26_214_400` works)

**Review by microwavenby — APPROVED** [review]

File: infra/app/envs/staging/main.tf (hunk 1 above)
Comment (rocketnova) [reply infra/app/envs/staging/main.tf]: Good call. Thank you. I'll change that.

File: infra/app/env-template/variables.tf (hunk 2 above)
Comment (rocketnova) [reply infra/app/env-template/variables.tf]: 🤔 Good question. I thought it did, but now I'm second guessing myself. I'll double check.

**Comment by rocketnova** [comment]
I'll add an inline @TODO to note that we should tidy up the code so that terraform creates the bucket and remix doesn't call create bucket.

File: infra/app/env-template/variables.tf (hunk 2 above)
Comment (rocketnova) [reply infra/app/env-template/variables.tf]: You were correct. Javascript `parseInt("26_214_400")` turns into `26`. I've changed this to just `"26214400"` which `parseInt()` does handle correctly. Good catch. Thank you!

**Pull request merged**